client/components/domains/accordion/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './style.scss';
```

Comment:
I think we [tend](https://github.com/Automattic/wp-calypso/blob/ddef3d8018c08476974f53d929ead7eb6ae8ea61/client/blocks/site-address-changer/index.jsx#L29) [to](https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/blocks/post-likes/index.jsx#L11) [keep](https://github.com/Automattic/wp-calypso/blob/2454d8bcea62fa08589a680f3ddfa4298a20f6b6/client/blocks/product-purchase-features-list/index.jsx#L49) the style import as the last one, although I'm not sure why the linter isn't correcting that...